src/QsFmt/Formatter/SyntaxTree/Statement.fsi

```diff
@@ -141,75 +175,65 @@ type internal Let =
     }
 
 /// <summary>
-/// A <c>return</c> statement.
+/// An <c>update</c> statement.
 /// </summary>
-type internal Return =
+type internal UpdateStatement =
     {
         /// <summary>
-        /// The <c>return</c> keyword.
+        /// The <c>set</c> keyword.
         /// </summary>
-        ReturnKeyword: Terminal
+        SetKeyword: Terminal
 
-        /// The returned expression.
-        Expression: Expression
+        /// The identifier being updated.
+        Name: Terminal
+
+        /// The update operator.
+        Operator: Terminal
+
+        /// The value used as the left-hand side of the update operator.
+        Value: Expression
 
         /// The semicolon.
         Semicolon: Terminal
     }
 
-/// The kind of qubit declaration.
-type internal QubitDeclarationKind =
-
-    /// <summary>
-    /// Indicates a <c>use</c> qubit declaration.
-    /// </summary>
-    | Use
-
-    /// <summary>
-    /// Indicates a <c>borrow</c> qubit declaration.
-    /// </summary>
-    | Borrow
-
-/// The concluding section of a qubit declaration.
-type internal QubitDeclarationCoda =
-
-    /// The semicolon.
-    | Semicolon of Terminal
-
-    /// The block of statements after the declaration.
-    | Block of Statement Block
-
-/// A qubit declaration statement.
-and internal QubitDeclaration =
+/// <summary>
+/// A <c>set w/=</c> statement.
+/// </summary>
+type internal SetWith =
```

Comment:
This naming to me just feels a bit inconsistent. Aligning closer with the UpdateStatement, I would consider calling it UpdateWithStatement.

Reply:
done